indico/modules/rb_new/operations/suggestions.py

```diff
@@ -100,10 +102,17 @@ def get_number_of_skipped_days_for_rooms(rooms, start_dt, end_dt, repeat_frequen
         if limit and len(data) == limit:
             break
 
+        suggestions = {}
+        if room.booking_limit_days is not None and room.booking_limit_days < booking_length:
```

Comment:
you need to take the global `booking_limit` into account if the room doesn't have one set:

booking_limit = room.booking_limit_days or rb_settings.get('booking_limit')